eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/services/BasicInsightsService.java

```diff
@@ -91,6 +91,15 @@ public void run()
 
 
     @Override
+    public int onStartCommand(Intent intent, int flags, int startId)
+    {
+        super.onStartCommand(intent, flags, startId);
```

Comment:
I think the calls to `super` are probably not needed.
(The onStartCommand() call in super is only for pre 2.0 compatibility.)